docs/guidelines/content-guidelines/04-ui-elements.md

```diff
@@ -0,0 +1,110 @@
+---
+title: UI elements
+---
+
+Contrary to the regular technical documentation, UI elements follow different guidelines and best practices. Due to the limited space you can use to convey a message, your UI text must be as precise and concise as possible. Here are some general guidelines you should follow when designing and writing content for UI elements:
+
+- Use active voice and action-oriented language.
+- Use title case for short one-line texts, such as labels or headings, and sentence case for messages and explanations, unless specified otherwise. (?)
+- Use plural for the object names, as users usually interact with multiple objects within the program, for example:
+
+  ✅ Service Instances <br>
+  ⛔️ Service Instance
+
+- Use plain, natural language. Even though some verbs may be similar in meaning, they may sound more or less natural to the reader. For example, from these pairs of verbs, the first one sound more natural:
+
+  ✅ Choose <br>
+  ⛔️ Select
+
+  ✅ Edit <br>
+  ⛔️ Modify
+
+- Use the three-letter currency code instead of the currency symbol. This prevents ambiguity when multiple currencies share the same symbol.
```

Comment:
Maybe we can add more info about symbols here? See [Fiori Guidelines](https://experience.sap.com/internal/fiori-design-web/ui-text-guidelines-for-sap-fiori/#symbols) for reference.

Reply:
Good point! Thx 👍